CIP-0100/README.md

```diff
@@ -180,8 +190,12 @@ The following properties are considered common to all types of transactions, and
   - `references`
     - An array of objects
     - Each object specifies a `@type`, which is either "GovernanceMetadata" or "Other"
+    - Each object specifies a unique integer `index` field value which increases without gaps from `0`. This is to act as a unique referable ID which can be mentioned in via other fields
     - Each object specifies a `label`, which serves as a human readable display name
     - Each object specifies a `uri`; when the type is set to "GovernanceMetadata", the URI should point to another CIP-0100 compliant document
+    - Each object MAY have a OPTIONAL `uriHash` object.
```

Comment:
> Make sure to add these to the context and schema.
Will do, but wanted to get confirmation on people's opinion on these changes first.

> We should also define a time where we're cutting off all further changes (barring mistakes / errors) to CIP-100; This could be a separate CIP which defines the context for this addition, for example, that was the whole point of using JSON-LD.
Yes, I did only make a PR because I don't believe there are any implementations of CIP-100 yet, so thought id try to sneak in these changes.